src/app/builders/BuilderFilterDropdown.tsx

```diff
@@ -15,11 +15,10 @@ export interface BuilderFilterDropdownProps {
 const builderFilterOptions = [
   { id: 'all', content: 'All Builders' },
   { id: 'active', content: 'Active Builders' },
-  { id: 'inactive', content: 'Inactive Builders' },
   { id: 'deactivated', content: 'Deactivated Builders' },
   { id: 'revoked', content: 'Revoked Builders' },
   { id: 'paused', content: 'Paused Builders' },
-  { id: 'in-progress', content: 'In Progress' },
+  { id: 'inProgress', content: 'In Progress' },
```

Comment:
We are missing the `selfPaused`, right now the builders self paused are included in the `inProgress`